phpunit.xml.dist

```diff
@@ -0,0 +1,35 @@
+<phpunit
+        bootstrap="tests/bootstrap.php"
+        backupGlobals="false"
+        colors="true"
+        beStrictAboutCoversAnnotation="true"
+        beStrictAboutOutputDuringTests="true"
+        beStrictAboutTestsThatDoNotTestAnything="true"
+        beStrictAboutTodoAnnotatedTests="true"
+        convertErrorsToExceptions="true"
+        convertNoticesToExceptions="true"
+        convertWarningsToExceptions="true"
+        verbose="true">
+
+    <testsuite name="beans">
+        <directory suffix=".php">./tests/</directory>
+    </testsuite>
+
+    <filter>
+        <whitelist processUncoveredFilesFromWhitelist="true">
+            <directory suffix=".php">.</directory>
+        </whitelist>
+    </filter>
+
+    <logging>
```

Comment:
That's a lot of logging. Is it really all needed?

Reply:
@GaryJones My bad...that was for my internal testing during the setup process.  Stripping it out everything by the `coverage-text`.